.eslintrc.js

```diff
@@ -2,12 +2,18 @@ module.exports = {
   root: true,
   parser: 'babel-eslint',
   env: {
+    'jest/globals': true,
     browser: true,
     node: true,
   },
-  extends: 'airbnb-base',
+  extends: [
+    'airbnb-base',
+    'plugin:jest/recommended',
+    'prettier',
```

Comment:
just found out about `plugin:vue/essential`, maybe want to add it also
https://www.npmjs.com/package/eslint-plugin-vue

Reply:
Let follow up on https://github.com/likecoin/puttyimages-web/issues/34